components/Starknet/modules/starknet_versions/pages/version_notes.adoc

```diff
@@ -16,13 +16,44 @@ include::ROOT:partial$snippet_important_goerli2_removed.adoc[]
 |Environment |Starknet version|Sierra version|Cairo version
 
 |Mainnet|0.13.0|1.4.0|2.0.0 - 2.5.4
-
-|Sepolia Testnet|0.13.0|1.4.0|2.0.0 - 2.5.4
+|Sepolia Testnet|0.13.1|1.5.0|2.0.0 - 2.6
 |Goerli Testnet (deprecated)|0.13.0|1.4.0|2.0.0 - 2.5.4
 |===
 
+[id="version0.13.1"]
+== Starknet v0.13.1
+
+[discrete]
+=== New features and enhancements
+
+* *Cheaper data availability:* Starknet uses link:https://eips.ethereum.org/EIPS/eip-4844[EIP-4844].
+* Time-related syscalls when called from an account contract's `__validate__` function:
+** `timestamp` returns the hour, rounded down.
+** `block_number` returns the block number, rounded down to the nearest multiple of 100.
+* *Optimization:* Load into memory only the functions in a contract that are actually used when generating the proof.
+* Add `starknet-compiled-class-hash` command.
```

Comment:
@ArielElp Add it to what?

Reply:
I'm removing this line. Lior explained that this is something specific to cairo-lang that he added to the cairo-lang repo, but it is not useful here.